docs/modules/ROOT/pages/CHANGELOG.adoc

```diff
@@ -1,5 +1,32 @@
 = Changelog
 
+[[release-0.6.0]]
+== 0.6.0 (2025-10-02)
+
+=== Fixes
+
+* Avoid unnecessary computations when debug logs are omitted (https://github.com/apple/pkl-swift/pull/60[#60]).
+* Avoid generating typealiases of hidden properties (https://github.com/apple/pkl-swift/pull/63[#63]).
+* Fix decoding of polymorphic types wihtin collections (https://github.com/apple/pkl-swift/pull/66[#66]).
+
+=== Additions
+
+* Adds support for Swift 6 (https://github.com/apple/pkl-swift/pull/61[#61], https://github.com/apple/pkl-swift/pull/68[#68]).
+* Adds support for decoding `Class` and `TypeAlias` values (https://github.com/apple/pkl-swift/pull/64[#64]).
+
+=== Breaking Changes 
+
+* The code generation of optional keys has changed. A `Mapping` with an optional key will be represented as `OptionalDictionaryKey<T>` in Swift. For example, `Mapping<String?, String>` becomes `[OptionalDictionaryKey<String>: String]` in Swift (https://github.com/apple/pkl-swift/pull/66[#66]).
```

Comment:
This applies to `Map` as well.

Also, this is technically only breaking to code that was already broken! Prior to this change, the old generated code never successfully decoded from the binary format so I don't expect affected code to actually exist in the wild.